accounting/account/models.py

```diff
@@ -117,6 +117,9 @@ class HourlyRate(CommonModel, UuidModel):
     client = models.ForeignKey(
         Account, models.CASCADE, related_name='client_hourly_rates', blank=True, null=True,
         help_text=_("The account linked to the user who pays the hourly rate to the provider."))
+    active = models.BooleanField(
```

Comment:
This field allows us to more easily deal with tracking whether someone is still under contract or not. I tried doing that through checking if JIRA worklogs existed, but it complicated things a lot more and gave rise to new edge cases, so I dropped it in favor of this.

Reply:
@UmanShahzad Sounds good - that does sound better yep.